watson_developer_cloud/discovery_v1.py

```diff
@@ -226,6 +226,29 @@ def create_collection(self,
                             params={'version': self.version},
                             accept_json=True)
 
+    def update_collection(self,
+                          environment_id,
+                          collection_id,
+                          name,
+                          description="",
+                          configuration_id=None):
+
+        if configuration_id is None:
```

Comment:
Sorry, just got to this pull request. Does this default configuration_id logic already live within the service? (When absent, doesn't the configuration_id default to the default configuration_id?)